build/rust/Cargo.lock

```diff
@@ -1403,9 +1402,9 @@ checksum = "60302e4db3a61da70c0cb7991976248362f30319e88850c487b9b95bbf059e00"
 
 [[package]]
 name = "memchr"
-version = "2.4.1"
+version = "2.5.0"
```

Comment:
@bridiver requested we stay on `2.4.1` here so we can use Chromium's vendored version. That seems reasonable to me looking at the changes between the two releases. Do you mind downgrading the requirement on the adblock-rust Cargo.toml and updating here?

Reply:
done